docs/private-registries.md

```diff
@@ -0,0 +1,35 @@
+# Ratify with Private Container Registries
+The default content store that is configured with Ratify is a container registry that is queried using the  ```oras``` store extension. This document outlines options to authenticate with the private container registries for querying supply chain content like signatures. 
+
+As of now, ```oras``` supports two ways to authenticate with the private registries. 
+- Providing Username and Password for the registry
+- Retrieving the registry credentials from Docker Config 
+
+> In future, ```oras``` will provide abstractions for authentication that enables providing credentials through other options like k8s secret, credential provider etc. 
+
+For CLI mode of Ratify, the authentication should work automatically by doing ```docker login``` to the private registry. The below section describes the steps to configure this authentication for the k8s deployment of Ratify.
+
+## Authentication using Docker credentials
+
+- Create a k8s secret by providing credentials on the command line. This secret should be in the same namespace that contains Ratify deployment. 
+
+```bash
+kubectl create secret docker-registry ratify-regcred --docker-server=<your-registry-server> --docker-username=<your-name> --docker-password=<your-pword> --docker-email=<your-email>
+```
+ 
+- Deploy Ratify using helm
+```bash
+helm install ratify charts/ratify --set registryCredsSecret=ratify-regcred
```

Comment:
Do you expect this to expand to support multiple secrets at some point. I think we can track that as another issue.

Reply:
Yes, to address in this PR, I updated the doc to clarify the same.